test/queue.js

```diff
@@ -4,13 +4,10 @@ var assert = require('assert');
 
 
 describe('queue', function(){
-    // several tests of these tests are flakey with timing issues
-    this.retries(3);
-
     it('basics', (done) => {
 
         var call_order = [];
-        var delays = [40,10,60,10];
+        var delays = [60,10,100,10];
```

Comment:
Similar to my earlier comment, can you increase each value more consistently -- i.e. same multiplier. So for example: `80,20,120,20`